src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/OveruseEstimator.java

```diff
@@ -240,6 +260,14 @@ public void update(
         slope = slope + K[0] * residual;
         prevOffset = offset;
         offset = offset + K[1] * residual;
+
+        /**
+         * Migrated from webrtc. Used for logging
+         */
+//        BWE_TEST_LOGGING_PLOT(1, "kc", nowMs, K[0]);
+//        BWE_TEST_LOGGING_PLOT(1, "km", nowMs, K[1]);
+//        BWE_TEST_LOGGING_PLOT(1, "slope_1/bps", nowMs, slope);
+//        BWE_TEST_LOGGING_PLOT(1, "var_noise", nowMs, varNoise_);
```

Comment:
I don't think we need any of the changes to this file -- all usage of nowMs is commented out.